manifests/e/Eigenmiao/Rickrack/2.7.23/Eigenmiao.Rickrack.locale.en-US.yaml

```
@@ -0,0 +1,25 @@
+# Created using wingetcreate 1.2.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.1.0.schema.json
+
+PackageIdentifier: Eigenmiao.Rickrack
+PackageVersion: 2.7.23
+PackageLocale: en-US
+Publisher: Eigenmiao
+PublisherUrl: https://eigenmiao.com/rickrack/
+PublisherSupportUrl: https://eigenmiao.com/yanhuo/en.html
+PackageName: Rickrack
+License: GNU General Public License 3.0 (GPL 3.0)
+LicenseUrl: https://www.gnu.org/licenses/
+Copyright: Copyright (c) 2019-2022 Eigenmiao. All Rights Reserved.
+ShortDescription: Generate Harmonious Colors Freely.
+Description: Rickrack (Real-time Color Kit) is a free color editor. It is designed for generating a set of harmonious colors from color wheel or elsewhere. You can share these colors with your friends, or apply them into your creative works.
+Tags:
+- color-palette
+- color-editor
+- color-picker
+- color-generator
+- color-wheel
+- Rickrack

```

```suggestion
- rickrack
```
As per the schema docs, tags should be generally written in lowercase